imap_processing/swapi/swapi_metadata.json

```diff
@@ -0,0 +1,246 @@
+            "FIELDNAM": "Secondary CEM uncertainty",
+            "FORMAT": "E19.5",
+            "LABLAXIS": "SCEM uncert",
+            "UNITS": "counts",
+            "VALIDMIN": 0.0,
+            "VALIDMAX": 1.7976931348623157e+308,
+            "FILLVAL": -9.22337e+18,
+            "SCALE_TYP": "linear"
+        },
+        "swp_coin_err": {
+            "CATDESC": "Coincidence uncertainty",
+            "VAR_TYPE": "data",
+            "DEPEND_0": "epoch",
+            "DEPEND_1": "energy",
+            "DISPLAY_TYPE": "spectrogram",
+            "FIELDNAM": "Coincidence uncertainty",
+            "FORMAT": "E19.5",
+            "LABLAXIS": "COIN uncert",
+            "UNITS": "counts",
+            "VALIDMIN": 0.0,
+            "VALIDMAX": 1.7976931348623157e+308,
+            "FILLVAL": -9.22337e+18,
+            "SCALE_TYP": "linear"
+        }
+    },
+    "imap_swapi_l1_hk": {
+        "global_attrs": {
+            "Logical_file_id": "<to-be-replaced-in-code>",
+            "Data_type": "L1A_HK>Level-1A Housekeeping Data",
+            "Logical_source": "imap_swapi_l1_hk",
+            "Logical_source_description": "SWAPI Instrument Level-1 Housekeeping Data",
+            "Project": "STP>Solar-Terrestrial Physics",
+            "Source_name": "IMAP>Interstellar Mapping and Acceleration Probe",
+            "Discipline": "Solar Physics>Heliospheric Physics",
+            "Mission_group": "IMAP>Interstellar Mapping and Acceleration Probe",
+            "PI_name": "Dr. David J. McComas",
+            "PI_affiliation": "Princeton Plasma Physics Laboratory",
+            "File_naming_convention": "source_descriptor_datatype_yyyyMMdd_vNNN",
+            "Data_version": 1,
+            "Descriptor": "swapi>The Solar Wind and Pickup Ion",
+            "TEXT": "The Solar Wind and Pickup Ion (SWAPI) instrument measures several different elements of the solar wind, including hydrogen (H) and helium (He) ions, and, on occasion, heavy ions produced by large events from the Sun. See https://imap.princeton.edu/instruments/swapi for more details. SWAPI level-1 data contains primary, secondary, coincidence counts per ESA voltage step and time. Level-2 data contains the same data as level-1 but counts are converted to rates by dividing counts by time.",
+            "Instrument_type": "Particles (space)"
+        },
+        "epoch": {
+            "CATDESC": "Time, number of nanoseconds since J2000 with leap seconds included",
+            "FIELDNAM": "epoch",
+            "LABLAXIS": "epoch",
+            "UNITS": "ns",
+            "VALIDMIN": -3.15576e+17,
+            "VALIDMAX": 3.15563e+18,
+            "FILLVAL": -9.22337e+18,
+            "VAR_TYPE": "support_data",
+            "SCALE_TYP": "linear",
+            "TIME_BASE": "J2000",
+            "TIME_SCALE": "Terrestrail Time",
+            "MONOTON": "INCREASE",
+            "REFERENCE_POSITION": "Rotating Earth Geoid"
+        },
+        "<to-be-replaced-in-code>": {
```

Comment:
It is probably worth coming up with a standard syntax for items that will be replaced in code.